src/renderer/library/components/dialog/DeletePublicationConfirm.tsx

```diff
@@ -38,13 +38,19 @@ class DeletePublicationConfirm extends React.Component<IProps, undefined> {
         super(props);
 
         this.remove = this.remove.bind(this);
+
+        this.yesButtonRef = React.createRef();
     }
 
+    private yesButtonRef: React.RefObject<HTMLButtonElement>;
+
     public render(): React.ReactElement<{}> {
         if (!this.props.open || !this.props.publicationView) {
             return <></>;
         }
 
+        setTimeout(() => this.yesButtonRef.current.focus(), 0);
```

Comment:
This is incorrect React code: first of all, `this.yesButtonRef.current` can be null, but more importantly `render()` only constructs the VDOM and may in fact be called multiple times. There is no guarantee that the actual DOM is ready at that point (i.e. at the `setTimeout()` tick). That's what a functional component's `useEffect()` is for (or class component equivalent).

I am working on a solution that works for all possible dialogs in Thorium (we need a consistent logic for focus management, including returning to the calling site).